src/commands/publish/mod.rs

```diff
@@ -120,11 +129,20 @@ pub fn upload_buckets(target: &Target, user: &GlobalUser) -> Result<(), failure:
                     path.display()
                 )
             }
-            kv::bucket::sync(target, user.to_owned(), &namespace.id, path, false)?;
+            let manifest_result =
+                kv::bucket::sync(target, user.to_owned(), &namespace.id, path, false)?;
+            if target.site.is_some() {
+                if asset_manifest.is_none() {
+                    asset_manifest = Some(manifest_result)
+                } else {
+                    // only site manifest should be returned
+                    unreachable!()
```

Comment:
can we just get rid of this else clause entirely?

Reply:
Actually I think this logic is flawed.. What happens when there are buckets side by side with a site. I think that this should check `target.site.bucket` and see if it's the same as the current bucket in the loop